javascript/javascript1/week1/js-week1-homework/script.js

```diff
@@ -1,81 +1,112 @@
-    let juliaHousePrice = juliaHouseVolume * salesPriceRate * houseUnitPrice;
-    //ราคาขายรวม
-    let peterSalePrice = peterHousePrice + peterGardenPrice;
-    let juliaSalePrice = juliaHousePrice + juliaGardenPrice;
-    //ผลต่าง
-    let peterDiff = peterSalePrice - peterHouseCost;
-    let juliaDiff = juliaSalePrice - juliaHouseCost;
+  let peterHousePrice = peterHouseVolume * salesPriceRate * houseUnitPrice;
+  let juliaHousePrice = juliaHouseVolume * salesPriceRate * houseUnitPrice;
 
-    let logString = "Peter's price=" + peterHousePrice + " Cost=" + peterHouseCost + " Change=" + peterDiff;
-    logString += "\nJulia's price=" + juliaHousePrice + " Cost=" + juliaHouseCost + " Change=" + juliaDiff;
+  let peterSalePrice = peterHousePrice + peterGardenPrice;
+  let juliaSalePrice = juliaHousePrice + juliaGardenPrice;
 
-    alert(logString);
-    console.log(logString);
+  let peterDiff = peterSalePrice - peterHouseCost;
+  let juliaDiff = juliaSalePrice - juliaHouseCost;
+
+  let logString =
+    "Peter's price=" +
+    peterHousePrice +
+    " Cost=" +
+    peterHouseCost +
+    " Change=" +
+    peterDiff;
+  logString +=
+    "\nJulia's price=" +
+    juliaHousePrice +
+    " Cost=" +
+    juliaHouseCost +
+    " Change=" +
+    juliaDiff;
+
+  alert(logString);
+  console.log(logString);
 }
 
 function getStartupName() {
-    //first words array for first name of the company
-    const firstWords = ["Eazy", "Juicy", "Awezome", "Superb", "Exzellent", "YeahYeah", "Nicey", "Perrfect", "Alle", "ForYou"];
-    //second words array for type of the company
-    const secondWords = ["Company", "Solution", "Technology", "Soft", "Cooperation", "Inc", "Limited", "Corp", "Partnership", "Application"];
-    //find random value with in total of records in array
-    let randomFirst = Math.floor(Math.random() * firstWords.length);
-    let randomSecond = Math.floor(Math.random() * secondWords.length);
-    //create company name
-    let startupName = firstWords[randomFirst] + " " + secondWords[randomSecond];
-    //answer result
-    let logString = 'The Startup: "' + startupName + '"';
-    alert(logString);
-    console.log(logString);
-}
\ No newline at end of file
+  //first words array for first name of the company
+  const firstWords = [
+    "Eazy",
+    "Juicy",
+    "Awezome",
+    "Superb",
+    "Exzellent",
+    "YeahYeah",
+    "Nicey",
+    "Perrfect",
+    "Alle",
+    "ForYou",
+  ];
+  //second words array for type of the company
+  const secondWords = [
+    "Company",
+    "Solution",
+    "Technology",
+    "Soft",
+    "Cooperation",
+    "Inc",
+    "Limited",
+    "Corp",
+    "Partnership",
+    "Application",
+  ];
+  //find random value with in total of records in array
+  let randomFirst = Math.floor(Math.random() * firstWords.length);
+  let randomSecond = Math.floor(Math.random() * secondWords.length);
+  //create company name
+  let startupName = firstWords[randomFirst] + " " + secondWords[randomSecond];
```

Comment:
Very good job on using random indexes! And again, great code formatting, logic and comments! 💪